pkg/ccl/importccl/exportcsv.go

```diff
@@ -72,8 +72,7 @@ func exportPlanHook(
 	}
 
 	if exportStmt.FileFormat != "CSV" {
-		// not possible with current parser rules.
-		return nil, nil, nil, errors.Errorf("unsupported import format: %q", exportStmt.FileFormat)
+		return nil, nil, nil, errors.Errorf("unsupported export format: %q", exportStmt.FileFormat)
```

Comment:
I think this is an unrelated change that deserves a separate commit (either in this pr or another)

Reply:
moved to a second commit in this pr